web/modules/custom/dpl_admin/src/Controller/AdminPages.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\dpl_admin\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Url;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Custom admin pages.
+ */
+class AdminPages extends ControllerBase {
+
+  /**
+   * The entity type interface.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container): AdminPages|static {
+    return new static(
+      $container->get('entity_type.manager'),
+    );
+  }
+
+  /**
+   * The 'add content' overview page, displaying options for the editors.
+   *
+   * @return array<mixed>
+   *   A render array, for displaying the page.
+   */
+  public function addContentPage(): array {
+
+    // 'Create from scratch' links - used in the admin block below.
+    $create_links = [
+      [
+        'title' => $this->t('Event series', [], ['context' => 'DPL admin UX']),
+        'url' => Url::fromRoute('entity.eventseries.add_page'),
+        'description' => $this->t('Use event series to create event instances - you can use series, both to create reoccurring events, and single events.', [], ['context' => 'DPL admin UX']),
+      ],
+    ];
+
+    $node_types = $this->entityTypeManager->getStorage('node_type')->loadMultiple();
```

Comment:
Is it possible to also load the `eventseries` from the Entity Type Manager? Perhaps you opted not to do so because we use the `default` type which does not render very well. 🤔

Reply:
Exactly :)
Events are pretty much outliers, so I wanted a custom link button.

If we one day end up with another type other than `default`, this button will still work, as it links to the "all types" page.